eva/udfs/yolo_object_detector.py

```diff
@@ -0,0 +1,204 @@
+        return [
+            "__background__",
+            "person",
+            "bicycle",
+            "car",
+            "motorcycle",
+            "airplane",
+            "bus",
+            "train",
+            "truck",
+            "boat",
+            "traffic light",
+            "fire hydrant",
+            "N/A",
+            "stop sign",
+            "parking meter",
+            "bench",
+            "bird",
+            "cat",
+            "dog",
+            "horse",
+            "sheep",
+            "cow",
+            "elephant",
+            "bear",
+            "zebra",
+            "giraffe",
+            "N/A",
+            "backpack",
+            "umbrella",
+            "N/A",
+            "N/A",
+            "handbag",
+            "tie",
+            "suitcase",
+            "frisbee",
+            "skis",
+            "snowboard",
+            "sports ball",
+            "kite",
+            "baseball bat",
+            "baseball glove",
+            "skateboard",
+            "surfboard",
+            "tennis racket",
+            "bottle",
+            "N/A",
+            "wine glass",
+            "cup",
+            "fork",
+            "knife",
+            "spoon",
+            "bowl",
+            "banana",
+            "apple",
+            "sandwich",
+            "orange",
+            "broccoli",
+            "carrot",
+            "hot dog",
+            "pizza",
+            "donut",
+            "cake",
+            "chair",
+            "couch",
+            "potted plant",
+            "bed",
+            "N/A",
+            "dining table",
+            "N/A",
+            "N/A",
+            "toilet",
+            "N/A",
+            "tv",
+            "laptop",
+            "mouse",
+            "remote",
+            "keyboard",
+            "cell phone",
+            "microwave",
+            "oven",
+            "toaster",
+            "sink",
+            "refrigerator",
+            "N/A",
+            "book",
+            "clock",
+            "vase",
+            "scissors",
+            "teddy bear",
+            "hair drier",
+            "toothbrush",
+        ]
+
+    def forward(self, frames: Tensor) -> pd.DataFrame:
+        """
+        Performs predictions on input frames
+        Arguments:
+            frames (np.ndarray): Frames on which predictions need
+            to be performed
+
+        Returns:
+            tuple containing predicted_classes (List[List[str]]),
+            predicted_boxes (List[List[BoundingBox]]),
+            predicted_scores (List[List[float]])
+
+        """
+        # img = frames[0].transpose()
+        
+        img_b = frames[0][0]
+        img_g = frames[0][1]
+        img_r = frames[0][2]
+        img_rgb = torch.stack((img_r, img_g, img_b), dim=2)
+
+        img_rgb = img_rgb.cpu().detach().numpy()
+        img_rgb = img_rgb*255
+        print(type(img_rgb))
+
+        predictions = self.model(img_rgb)
+        predictions.print()
```

Comment:
Please remove the print statements.

Reply:
Thank you, for pointing that out, will remove that.